packages/flutter/lib/src/cupertino/context_menu_action.dart

```diff
@@ -39,11 +41,11 @@ class CupertinoContextMenuAction extends StatefulWidget {
   /// Called when the action is pressed.
   final VoidCallback onPressed;
 
-  /// An optional icon to display to the right of the child.
+  /// An optional [Icon] widget to display to the right of the child.
   ///
   /// Will be colored in the same way as the [TextStyle] used for [child] (for
   /// example, if using [isDestructiveAction]).
-  final IconData trailingIcon;
+  final Widget trailing;
```

Comment:
This would be a rally big breaking change for anyone who uses this property. Usually we deprecate before removing properties to avoid such a break, you can read about it in our [breaking changes policy](https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes).